client/components/activity-card-list/index.jsx

```diff
@@ -293,7 +306,6 @@ class ActivityCardList extends Component {
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
 			<div className="activity-card-list__loading-placeholder">
-				<div className="filterbar" />
```

Comment:
We don't need this anymore, because we are using the `isLoading` prop of `Filterbar` now.